tests/integration/utils.py

```diff
@@ -189,6 +190,16 @@ def __check_attributes_schema(actual_attributes, expected_attributes):
             expected_attribute = expected_attributes[field]
             expected_type = __get_attribute_type(expected_attribute)
 
+            # Check item schema if attribute is an array
+            if expected_type is list and \
```

Comment:
As discussed, the `expected_type` will be the real type in Python instead of the type in the openapi definition so it should be `list`. As for the accurate operation to compare the type, I think using `is` and `==` are both correct and both are used by people. I also found this [article](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance) says that using `is` is more appropriates than `==`:

> ```
> type(obj) is Base
> ```
> 
> only supports the type referenced.
> 
> As a sidenote, `is` is likely more appropriate than
> 
> ```
> type(obj) == Base
> ```
> 
> because classes are singletons.

So I'll keep the way I implemented as now.